tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/Checks/AgentConnectivityCheckTests.cs

```diff
@@ -43,17 +47,60 @@ public async Task DetectAgentUrl((string, string)[] environmentVariables)
 
             processInfo.Should().NotBeNull();
 
-            var result = await AgentConnectivityCheck.Run(processInfo);
+            _ = await AgentConnectivityCheck.Run(processInfo!);
 
             console.Output.Should().Contain(DetectedAgentUrlFormat("http://fakeurl:7777/"));
         }
 
+        [Fact]
+        public async Task DetectTransportHttp()
+        {
+            using var agent = new MockTracerAgent(TcpPortProvider.GetOpenPort());
+
+            var url = $"http://127.0.0.1:{agent.Port}/";
+
+            using var helper = await StartConsole(enableProfiler: false, ("DD_TRACE_AGENT_URL", url));
+
+            using var console = ConsoleHelper.Redirect();
+
+            var processInfo = ProcessInfo.GetProcessInfo(helper.Process.Id);
+
+            processInfo.Should().NotBeNull();
+
+            _ = await AgentConnectivityCheck.Run(processInfo!);
+
+            console.Output.Should().Contain(ConnectToEndpointFormat(url, "HTTP"));
+        }
+
+#if NETCOREAPP3_1_OR_GREATER
+        [Fact]
+        public async Task DetectTransportUds()
```

Comment:
Just noticed we don't have the `RunOnWindows` trait set on any of these tests. These _should_ be running on Windows right?

> As an aside, I think we should run everything on Windows by default, and require you to explicitly set `RunOnWindows` to `false` to escape that

Reply:
While some later windows versions support UDS for Stream/IP, I don't think the agent even considers it as an option.
That said, I'm not against verifying and being ahead of the game.

Reply:
> While latest windows versions support UDS for Stream/IP, I don't think the agent even considers it as an option.

cc @pierotibou, something to bear in mind/double check for the game day

Reply:
> While latest windows versions support UDS for Stream/IP, I don't think the agent even considers it as an option.

It does, I tested it 🙂